src/components/user/createAppointment/checkboxes/Checkboxes.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Checkbox from './Checkbox';
+
+const Checkboxes = (props) => (
+  <>
+    <Checkbox time="12:00 am" onChangeTime={(time) => props.onChangeTime(time)} />
+    <Checkbox time="1:00 pm" onChangeTime={(time) => props.onChangeTime(time)} />
+    <Checkbox time="2:00 pm" onChangeTime={(time) => props.onChangeTime(time)} />
+    <Checkbox time="3:00 pm" onChangeTime={(time) => props.onChangeTime(time)} />
+    <Checkbox time="4:00 pm" onChangeTime={(time) => props.onChangeTime(time)} />
+    <Checkbox time="5:00 pm" onChangeTime={(time) => props.onChangeTime(time)} />
+    <Checkbox time="6:00 pm" onChangeTime={(time) => props.onChangeTime(time)} />
+    <Checkbox time="7:00 pm" onChangeTime={(time) => props.onChangeTime(time)} />
+    <Checkbox time="8:00 pm" onChangeTime={(time) => props.onChangeTime(time)} />
+    <Checkbox time="9:00 pm" onChangeTime={(time) => props.onChangeTime(time)} />
```

Comment:
Wow, create array of time ['12:00 am', .... and use map, code duplicates its kinda bad idea

Reply:
```
const data = [
  '12:00 am',
  '1:00 pm',
  '2:00 pm',
  '3:00 pm',
  '4:00 pm',
  '5:00 pm',
  '6:00 pm',
  '7:00 pm',
  '8:00 pm',
  '9:00 pm',
]

<Listing onClick={handleClick}>
        {data.map((item) => (
          <li key={item}>
            <Button>{item}</Button>
          </li>
        ))}
      </Listing>
```

Reply:
@Lemout17 good point.